src/plugins/filter/demo/post-filter.js

```diff
@@ -0,0 +1,17 @@
+( function() {
+window[ "wb-filter" ] = {
+	filterCallback: function( $field, $elm, settings ) {
+		var $sections =	$elm.find( "section" ),
```

Comment:
You can still allow to use a custom callback for those complex case. 

Is triggering a event could be better than using a call back?